modules/preparing-aws-sts-credentials-for-oadp.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-aws-sts/oadp-aws-sts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-aws-sts-credentials-for-oadp_{context}"]
+= Preparing AWS STS credentials for OADP
+
+An {aws-full} account must be prepared and configured to accept an {oadp-first} installation. Prepare the {aws-short} credentials by following the proceeding steps.
+
+.Procedure
+. Define the `cluster_name` environment variable by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME= <AWS cluster name> <1>
```

Comment:
See guidelines on replaceable values: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values 

```suggestion
$ export CLUSTER_NAME= <AWS_cluster_name> <1>
```